djangoproject/urls.py

```diff
@@ -6,11 +6,12 @@
     TokenVerifyView
 )
 
-from djangoproject.djangoapp.views import user
+from djangoproject.djangoapp.views import CurrentUserView, UserViewSet
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('login/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
     path('login/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
-    path('users/me/', user, name='user')
+    path('users/me', CurrentUserView.as_view(), name='current_user'),
+    path('users/<int:pk>', UserViewSet.as_view({'get': 'retrieve'}), name='user')
```

Comment:
Takodje obrati paznju kako su ovdje odradili kacenje odredjenog viewset-a unutar urlspatterns
https://www.django-rest-framework.org/api-guide/routers/#using-include-with-routers. Sa ovim pristup se jednostavno ibjegava explicitno definisanje ovako ruta koje se korsite i dobijas odmah sve rute na osnovu toga sta si implementirao unutar svoje klase

Reply:
Sad kada sam spojio klase, imam problem da ovo uradim jer imam tu custom putanju users/me. Kada bih hteo da ostane takva ista putanja, ne znam kako to da uradim pomocu router-a.

Reply:
Uspeo sam! :)

Reply:
Uspeo sam da napravim to pomocu router-a.